### PR TITLE
refactor: replace ed card with action card + translations

### PR DESCRIPTION
**Pull request by disaerna**
# Service portal - Fix broken education cards

## What

Fix broken self made action cards and replace it with action card from ui core library.

## Why
Cuz it's broken 😞 

## Screenshots / Gifs

Attach Screenshots / Gifs to help reviewers understand the scope of the pull request

## Checklist:

- [ ] I have performed a self-review of my own code
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] Formatting passes locally with my changes
- [ ] I have rebased against main before asking for a review

### CODE REVIEW

File: libs/service-portal/education-license/src/screens/EducationLicense/components/LicenseCards/LicenseCards.tsx [hunk 1]
```diff
@@ -74,30 +78,35 @@ const LicenseCards = () => {
...
-            }
+          <ActionCard
+            cta={{
+              label: mutationLoading
+                ? formatMessage({
+                    id: 'sp.education-career:downloading',
+                    defaultMessage: 'Innsiglun skjals í vinnslu…',
+                  })
+                : formatMessage({
+                    id: 'sp.education-career:send-file',
```
Comment (darrikonn) [inline libs/service-portal/education-license/src/screens/EducationLicense/components/LicenseCards/LicenseCards.tsx:89]: id should be education-license.

File: libs/service-portal/education-license/src/screens/EducationLicense/components/LicenseCards/LicenseCards.tsx [hunk 2]
```diff
@@ -149,11 +178,17 @@ const LicenseCards = () => {
...
+                  {formatMessage({
+                    id: 'sp.education-career:cancel',
+                    defaultMessage: 'Hætta við',
+                  })}
                 </Button>
                 <Button disabled={expiry <= 0}>
                   <a download href={href} target="_blank" rel="noreferrer">
-                    Hlaða niður
+                    {formatMessage({
+                      id: 'sp.education-career:download',
```
Comment (darrikonn) [inline libs/service-portal/education-license/src/screens/EducationLicense/components/LicenseCards/LicenseCards.tsx:189]: ditto on all ids.

**Review by darrikonn — CHANGES_REQUESTED** [review]
Nice work.

File: libs/service-portal/education-license/src/screens/EducationLicense/components/LicenseCards/LicenseCards.tsx (hunk 1 above)
Comment (disaerna) [reply libs/service-portal/education-license/src/screens/EducationLicense/components/LicenseCards/LicenseCards.tsx]: ✅

File: libs/service-portal/education-license/src/screens/EducationLicense/components/LicenseCards/LicenseCards.tsx (hunk 2 above)
Comment (disaerna) [reply libs/service-portal/education-license/src/screens/EducationLicense/components/LicenseCards/LicenseCards.tsx]: ✅

File: libs/service-portal/education-license/src/screens/EducationLicense/components/LicenseCards/LicenseCards.tsx [hunk 3]
```diff
@@ -124,23 +133,43 @@ const LicenseCards = () => {
...
+                    })}
                   </Text>
                 </Box>
                 {expiry > 0 ? (
                   <Text>
-                    Athugið að niðurhalslinkurinn rennur út eftir{' '}
+                    {formatMessage({
+                      id: 'sp.education-license:download-link-expiry',
+                      defaultMessage:
+                        'Athugið að niðurhalslinkurinn rennur út eftir',
```
Comment (kksteini) [inline libs/service-portal/education-license/src/screens/EducationLicense/components/LicenseCards/LicenseCards.tsx:156]: Er ekki betri íslenska: `niðurhalshlekkurinn`.

File: libs/service-portal/education-license/src/screens/EducationLicense/components/LicenseCards/LicenseCards.tsx [hunk 4]
```diff
@@ -124,23 +133,43 @@ const LicenseCards = () => {
...
+                      id: 'sp.education-license:seconds',
+                      defaultMessage: 'sekúndur.',
+                    })}
                   </Text>
                 ) : (
-                  <Text color="red400">Niðurhalslinkurinn er útrunninn</Text>
+                  <Text color="red400">
+                    {formatMessage({
+                      id: 'sp.education-license:download-link-expired',
+                      defaultMessage: 'Niðurhalslinkurinn er útrunninn.',
```
Comment (kksteini) [inline libs/service-portal/education-license/src/screens/EducationLicense/components/LicenseCards/LicenseCards.tsx:170]: Sama hér

**Review by kksteini — APPROVED** [review]

**Pull request merged**